doc/api/http2.md

```diff
@@ -3935,6 +3935,16 @@ const server = http2.createServer((req, res) => {
 });
 ```
 
+#### `response.setHeaders()`
+
+* `headers` {Headers|Map}
+
+Sets multiple header values for implicit headers.
+`headers` must be an instance of [`Headers`][] or `Map`,
+if a header already exists in the to-be-sent headers,
```

Comment:
```suggestion
`headers` must be an instance of [`Headers`][] or `Map`.
If a header already exists in the to-be-sent headers,
```

Reply:
Thanks for the feedback :raised_hands: 
Fixed [here](https://github.com/nodejs/node/pull/51576/commits/bf256464d574d6318d1165dec000660f84b62fba)